shared/test/render-dumb-sheet.js

```diff
@@ -9,44 +9,62 @@ import DumbSheetItem from '../dev/dumb-sheet/item'
 
 const PADDING = 25
 
-ipcRenderer.on('display', (ev, msg) => {
-  const map = dumbComponentMap[msg.key]
-  const mockKey = msg.mockKey
+window.load = (options) => {
```

Comment:
I was holding off from adding this unless it was absolutely necessary -- I don't think it's required, since the supervising code waits for the window to load, but it probably doesn't hurt either.